src/main/java/com/mongodb/hibernate/jdbc/ConnectionAdapter.java

```diff
@@ -0,0 +1,306 @@
+/*
+ * Copyright 2024-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.hibernate.jdbc;
+
+import java.sql.*;
```

Comment:
`*` imports keep appearing in code changes. Given that it's impossible to forbid them with the current linter, could you please change your IDE formatting settings such that it never does `*` imports?

Reply:
sorry for the annoyance. The issue is .editorconfig overrides IDE setting and in this branch we didn't sync up with main branch (we fixed it at https://github.com/mongodb/mongo-hibernate/pull/16/files#r1847076715). For now I have to keep it in mind and manually fix it.

Reply:
I updated the base branch of this PR so the issue should not trouble me next time.